cluster-scope/overlays/nerc-ocp-infra/kustomization.yaml

```diff
@@ -22,6 +22,7 @@ resources:
 - clusterrolebindings/nerc-ops-sudoers.yaml
 - clusterrolebindings/nerc-ops-portforward.yaml
 - externalsecrets
+- apiserver/api_server_cert.yaml
```

Comment:
It's important to note that this isn't just setting the API certificate: it's unilaterally defining what the cluster APIServer resource looks like. This is why in #38 we create a `IngressController` resource in `base`, and then *patch* in the cluster overlay (so that we're clearly separating "this is what the resource should look like" from "this file just adds a certificate").

I'm not going to insist we do it the same way here, but we should rename this file to simply use the name of the resource it is creating, so `apiserver/cluster.yaml`.

Reply:
I appreciate the insight @larsks I've updated the filename as requested.